### ISSUE TITLE
Bug/Feature Request: I can't select the text when templates overlaps

### ISSUE THREAD

**Issue by mariorodriguespt** [bitflak/Meteor-Xray#3]
When combining templates, example: simple list view, It's hard to read the name of the template. A tree view would solve this I think.

**Comment by bitflak** [comment]
Good point! I will implement it but later. Sorry don't have the time right now.

**Comment by dandv** [comment]
@mariorodriguespt: check out the original package, of which this is a fork. https://github.com/Slava/Meteor-Xray/
